pkgdb/src/realisepkgs/realisepkgs-lockfile.cc

```diff
@@ -168,6 +170,11 @@ realisepkgsPackageFromV1Descriptor( const nlohmann::json &     jfrom,
       pkg.input       = nixpkgsHttpsToGithubInput( locked_url );
       pkg.input.attrs = flox::githubAttrsToFloxNixpkgsAttrs( pkg.input.attrs );
       pkg.input.url   = nix::FlakeRef::fromAttrs( pkg.input.attrs ).to_string();
+
+      for ( auto [output, outpath] : jfrom["outputs"].items() )
```

Comment:
Will all locked entities have outputs?  Like flakes and such?  Or should this be a conditional and later run the eval if there are no outputs in the locked package?

Reply:
Worth mentioning that this will need to honor `outputsToInstall` soon enough, so while simply enumerating `outputs` works for today it's not going to fly in the long run.

Reply:
> Will all locked entities have outputs? 

Yes - all locked entities (derivations) will have outputs which each bear a storePath.

Reply:
Probably worth making sure that mechanism works... including/testing it in publish, load from lockfile and collection in the realise command.